### PR TITLE
Add a thesis from 2016.

### PR DESCRIPTION
**Pull request by bangerth**
Not sure exactly how this is going to come out -- let's see.

### CODE REVIEW

**Comment by masterleinad** [comment]
We probably need to translate it.

**Comment by bangerth** [comment]
I don't even know what language this is in :-(

File: publications-2016.bib [hunk 1]
```diff
@@ -2132,4 +2132,16 @@ @misc{gassmoeller2016flexible
     primaryClass={math.NA}
 }
 
+
+@mastersthesis{AL:U0001-0508201623573200,
+title ={以相場法模擬冷凍鑄造法合成之仿生材料微結構},
```
Comment (masterleinad) [inline publications-2016.bib:2137]:
```suggestion
title ={Computational Phase-Field Modeling for Microstructural Evolution in Bioinspired Material from Freeze-Casting Process},
```

File: publications-2016.bib [hunk 2]
```diff
@@ -2132,4 +2132,16 @@ @misc{gassmoeller2016flexible
...
+author ={黃子軒},
```
Comment (masterleinad) [inline publications-2016.bib:2138]:
```suggestion
author ={Huang Zixuan},
```

File: publications-2016.bib [hunk 3]
```diff
@@ -2132,4 +2132,16 @@ @misc{gassmoeller2016flexible
...
+year ={2016},
+month ={Jan},
+school ={國立臺灣大學},
```
Comment (masterleinad) [inline publications-2016.bib:2141]:
```suggestion
school ={National Taiwan University},
```

**Comment by bangerth** [comment]
Amazing!

**Pull request merged**